lib/src/polyline.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:math' as math;
+
+List<List<double>> decodePolyline(String polyline, {int precision = 5}) {
```

Comment:
Let's also use our `Position` type here instead of `List<double>`

Reply:
Wrapping the `Position` in a `LineString`  will limit the `decodePolyline` usages. I think it would be best to have it just decode the Polyline string and let the user decide what to do with the results.